src/main/resources/scipooldev.properties

```diff
@@ -2,7 +2,8 @@ SYNAPSE_OAUTH_CLIENT_ID=100055
 SYNAPSE_OAUTH_CLIENT_SECRET=ssm::/synapse-login-scipooldev/SynapseOauthClientSecret
 AWS_REGION=us-east-1
 SESSION_TIMEOUT_SECONDS=28800
-USER_CLAIMS=userid
+SESSION_NAME_CLAIMS=userid
+SESSION_TAG_CLAIMS=sub,userid,user_name
```

Comment:
We may also want `firstName` and `lastName` from userProfile to tag resources provisioned through Service Catalog, mostly as a convenience for IT.

Reply:
Done!